python/src/scipp/compat/mantid.py

```diff
@@ -326,7 +324,7 @@ def convertMDHistoWorkspace_to_dataset(md_histo):
         dims_used.append(sc_dim)
     data = sc.Variable(dims=dims_used,
                        values=md_histo.getSignalArray(),
-                       variances=md_histo.getErrorSquaredArray())
+                       variances=md_histo.getErrorSquaredArray(), unit=sc.units.counts)
```

Comment:
Is this always this unit?

Reply:
That's a good question. I suspect the answer is usually yes, but it would be possible to make a MDHistoWorkspace one with other units.

Reply:
I'm not sure we can properly trace that information though.

Reply:
I think the preference in Mantid is to perform any kind of normalisation of counts of the fly.

Reply:
I guess you are referring to counts-densities, but this is a more general issue. The main example is probably data that has been normalized, e.g., by dividing two workspaces with unit counts, resulting in something dimensionless.

Reply:
I don't see a way to reliably get that information as the MDHistoWorkpaces don't store it.

Reply:
:(